docs/version0.44.md

```diff
@@ -88,6 +89,12 @@ Earlier, field names in `J9BuildFlags` were based on names defined in `j9.flags`
 
 You can extend the DDR code, adding your own commands, by writing plug-ins. If the user plug-in code contains references to fields of `J9BuildFlags` to read the build flags in the system dump data, you must change references to use the names as specified in `j9cfg.h`.
 
+### Deletion of the files within the `/tmp/` folder by `systemd` controlled
+
+You can use the `-Dcom.ibm.tools.attach.fileAccessUpdateTime` system property to prevent Linux&reg; `systemd` from deleting the files within the `/tmp/` folder. By controlling the automatic deletion of the files by `systemd`, the long-running Attach API can continue to use the temporary files to operate. This system property enables Attach API updates to the control file access times at specific intervals.
```

Comment:
from deleting the files within the -> from deleting the Attach API control files within the

Reply:
By controlling the automatic deletion of the files by `systemd` -> By updating the Attach API control file access times to avoid deletion by `systemd`

Reply:
enables Attach API updates to -> enables Attach API to update